### PR TITLE
fix: revert http server example on the homepage

### PR DESCRIPTION
**Pull request by tplk**
Revert example changes since https://github.com/denoland/deno_std/pull/188 was reverted (https://github.com/denoland/deno_std/commit/57c9176b19bf4f4580466e088c249cbe9b145119).

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -108,8 +108,8 @@ <h2 id="example">Example <a href="#example">#</a></h2>
 const s = serve("0.0.0.0:8000");
```
Comment (ry) [inline website/index.html]: Thanks can you use `https://deno.land/x/std@0.2.11/http/server.ts` above so we don't hit this again?

Comment (tplk) [reply website/index.html]: sure 👍

**Comment by tplk** [comment]
@ry oh, don't merge this one just yet. Looks like 0.2.11 is the version with http server module redesign included. So it either change the example back and lock the version or I can wait until newer version w/ the revert commit is released.

**Comment by ry** [comment]
Ah right, Use v0.2.10

**Pull request merged**